view-user/src/pages/index.tsx

```diff
@@ -1,45 +1,69 @@
 import type { NextPage } from "next";
 import { useRouter } from "next/router";
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useCallback } from "react";
 import styles from "@/styles/Home.module.css";
-import { ja, en } from "@/locales";
 import { useSubscription } from "@apollo/client";
 import { SubscribeListNumbersDocument } from "@/type/graphql";
 import type { SubscribeListNumbersSubscription } from "@/type/graphql";
 import { Layout, Loading, NumberCardLarge, NumberCardList } from "@/components";
 
+const defaultBingoNumber = {
+  number: 0,
+  id: 0,
+  createdAt: new Date(),
+  updatedAt: new Date(),
+};
+
```

Comment:
```suggestion
type numbers = SubscribeListNumbersSubscription["numbers"];
```
毎回Subscription...はコード量多いのでまとめちゃいましょう